spec/fixtures/people_skills.yml

```diff
@@ -0,0 +1,28 @@
+# == Schema Information
+#
+# Table name: people_skills
+#
+#  id              :bigint(8)        not null, primary key
+#  person_id       :bigint(8)
+#  skill_id        :bigint(8)
+#  level           :integer
+#  interest        :integer
+#  certificate     :boolean          default(FALSE)
+#  core_competence :boolean          default(FALSE)
+#
+
+people-skill1:
```

Comment:
Brauch hier im naming doch gleich den skill und dier person welche du joinst. Dann ist es um eigenes praktischer die Fixtures in den Tests zu verwenden